EIPS/EIP-8.md

```diff
@@ -0,0 +1,148 @@
+### Title
+
+      EIP: 8
+      Title: devp2p Forward Compatibility Requirements for Homestead
+      Author: Felix Lange <felix@ethdev.com>
+      Status: Draft
+      Type: Standards Track
+      Layer: Networking
+      Created: 2015-12-18
+
+### Abstract
+
+This EIP introduces new forward-compatibility requirements for implementations of the
+devp2p Wire Protocol, the RLPx Discovery Protocol and the RLPx TCP Transport Protocol.
+
+### Specification
+
+Implementations of the devp2p Wire Protocol should ignore the version number of hello
+packets. If the remote hello packet contains a version greater than the current local
+version, the implementation should reply with a hello packet containing the local (lower)
+version. Implementations should also ignore any additional list elements at the end of the
+hello packet.
```

Comment:
I will clarify this in the text, it was definitely not meant to imply that implementations should wait for the remote packet.